testsuite/diem-fuzzer/src/lib.rs

```diff
@@ -93,3 +93,12 @@ pub fn fuzz_data_to_value<T: std::fmt::Debug>(
     let strategy_tree = strategy.new_tree(&mut runner).expect("should not happen");
     strategy_tree.current()
 }
+
+/// Bake lsan suppressions into the binary
+#[no_mangle]
+pub extern "C" fn __lsan_default_suppressions() -> *const c_char {
+    let s = CString::new(include_str!("../lsan_suppressions.txt")).unwrap();
+    let p = s.as_ptr();
+    std::mem::forget(s);
+    p
```

Comment:
Might be better to wrap this with a static [once_cell::sync::Lazy](https://docs.rs/once_cell/1.8.0/once_cell/sync/struct.Lazy.html) or in order to avoid needing to construct the Cstring and leak memory on each call

Reply:
@bmwill : The code is not the easiest to follow but I did confirm at runtime that `LeakSanitizer` only calls this function once.

The code also indicates this if you read through https://github.com/llvm/llvm-project/blob/d480f968ad8b56d3ee4a6b6df5532d485b0ad01e/compiler-rt/lib/lsan/lsan_common.cpp